test_testvectors.py: 15 AVX2 tests fail on macOS

macOS 10.15.6, Xcode11.7.

```
clang -mavx2 -O3 -Wall -Wconversion -Wextra -Wpedantic -Wvla -Werror -Wmissing-prototypes -Wredundant-decls -std=c99 -I../../../common  -mavx2 -c KeccakP-1600-times4-SIMD256.c -o KeccakP-1600-times4-SIMD256.o
KeccakP-1600-times4-SIMD256.c:158:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                Xor_In( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:138:66: note: expanded from macro 'Xor_In'
    #define Xor_In( argIndex )  XOReq256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:55: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~ ^
KeccakP-1600-times4-SIMD256.c:60:61: note: expanded from macro 'XOReq256'
    #define XOReq256(a, b)          a = _mm256_xor_si256(a, b)
                                                            ^
KeccakP-1600-times4-SIMD256.c:158:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                Xor_In( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:138:66: note: expanded from macro 'Xor_In'
    #define Xor_In( argIndex )  XOReq256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:68: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~              ^
KeccakP-1600-times4-SIMD256.c:60:61: note: expanded from macro 'XOReq256'
    #define XOReq256(a, b)          a = _mm256_xor_si256(a, b)
                                                            ^
KeccakP-1600-times4-SIMD256.c:158:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                Xor_In( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:138:66: note: expanded from macro 'Xor_In'
    #define Xor_In( argIndex )  XOReq256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:81: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~                           ^
KeccakP-1600-times4-SIMD256.c:60:61: note: expanded from macro 'XOReq256'
    #define XOReq256(a, b)          a = _mm256_xor_si256(a, b)
                                                            ^
KeccakP-1600-times4-SIMD256.c:158:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                Xor_In( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:138:66: note: expanded from macro 'Xor_In'
    #define Xor_In( argIndex )  XOReq256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:94: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~                                        ^
KeccakP-1600-times4-SIMD256.c:60:61: note: expanded from macro 'XOReq256'
    #define XOReq256(a, b)          a = _mm256_xor_si256(a, b)
                                                            ^
KeccakP-1600-times4-SIMD256.c:162:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                Xor_In( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:138:66: note: expanded from macro 'Xor_In'
    #define Xor_In( argIndex )  XOReq256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:55: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~ ^
KeccakP-1600-times4-SIMD256.c:60:61: note: expanded from macro 'XOReq256'
    #define XOReq256(a, b)          a = _mm256_xor_si256(a, b)
                                                            ^
KeccakP-1600-times4-SIMD256.c:162:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                Xor_In( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:138:66: note: expanded from macro 'Xor_In'
    #define Xor_In( argIndex )  XOReq256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:68: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~              ^
KeccakP-1600-times4-SIMD256.c:60:61: note: expanded from macro 'XOReq256'
    #define XOReq256(a, b)          a = _mm256_xor_si256(a, b)
                                                            ^
KeccakP-1600-times4-SIMD256.c:162:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                Xor_In( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:138:66: note: expanded from macro 'Xor_In'
    #define Xor_In( argIndex )  XOReq256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:81: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~                           ^
KeccakP-1600-times4-SIMD256.c:60:61: note: expanded from macro 'XOReq256'
    #define XOReq256(a, b)          a = _mm256_xor_si256(a, b)
                                                            ^
KeccakP-1600-times4-SIMD256.c:162:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                Xor_In( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:138:66: note: expanded from macro 'Xor_In'
    #define Xor_In( argIndex )  XOReq256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:94: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~                                        ^
KeccakP-1600-times4-SIMD256.c:60:61: note: expanded from macro 'XOReq256'
    #define XOReq256(a, b)          a = _mm256_xor_si256(a, b)
                                                            ^
KeccakP-1600-times4-SIMD256.c:167:13: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
            Xor_In( i );
            ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:138:66: note: expanded from macro 'Xor_In'
    #define Xor_In( argIndex )  XOReq256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:55: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~ ^
KeccakP-1600-times4-SIMD256.c:60:61: note: expanded from macro 'XOReq256'
    #define XOReq256(a, b)          a = _mm256_xor_si256(a, b)
                                                            ^
KeccakP-1600-times4-SIMD256.c:167:13: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
            Xor_In( i );
            ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:138:66: note: expanded from macro 'Xor_In'
    #define Xor_In( argIndex )  XOReq256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:68: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~              ^
KeccakP-1600-times4-SIMD256.c:60:61: note: expanded from macro 'XOReq256'
    #define XOReq256(a, b)          a = _mm256_xor_si256(a, b)
                                                            ^
KeccakP-1600-times4-SIMD256.c:167:13: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
            Xor_In( i );
            ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:138:66: note: expanded from macro 'Xor_In'
    #define Xor_In( argIndex )  XOReq256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:81: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~                           ^
KeccakP-1600-times4-SIMD256.c:60:61: note: expanded from macro 'XOReq256'
    #define XOReq256(a, b)          a = _mm256_xor_si256(a, b)
                                                            ^
KeccakP-1600-times4-SIMD256.c:167:13: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
            Xor_In( i );
            ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:138:66: note: expanded from macro 'Xor_In'
    #define Xor_In( argIndex )  XOReq256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:94: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~                                        ^
KeccakP-1600-times4-SIMD256.c:60:61: note: expanded from macro 'XOReq256'
    #define XOReq256(a, b)          a = _mm256_xor_si256(a, b)
                                                            ^
KeccakP-1600-times4-SIMD256.c:234:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                OverWr( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:214:66: note: expanded from macro 'OverWr'
    #define OverWr( argIndex )  STORE256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:55: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~ ^
KeccakP-1600-times4-SIMD256.c:56:70: note: expanded from macro 'STORE256'
    #define STORE256(a, b)          _mm256_store_si256((V256 *)&(a), b)
                                                                     ^
KeccakP-1600-times4-SIMD256.c:234:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                OverWr( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:214:66: note: expanded from macro 'OverWr'
    #define OverWr( argIndex )  STORE256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:68: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~              ^
KeccakP-1600-times4-SIMD256.c:56:70: note: expanded from macro 'STORE256'
    #define STORE256(a, b)          _mm256_store_si256((V256 *)&(a), b)
                                                                     ^
KeccakP-1600-times4-SIMD256.c:234:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                OverWr( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:214:66: note: expanded from macro 'OverWr'
    #define OverWr( argIndex )  STORE256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:81: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~                           ^
KeccakP-1600-times4-SIMD256.c:56:70: note: expanded from macro 'STORE256'
    #define STORE256(a, b)          _mm256_store_si256((V256 *)&(a), b)
                                                                     ^
KeccakP-1600-times4-SIMD256.c:234:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                OverWr( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:214:66: note: expanded from macro 'OverWr'
    #define OverWr( argIndex )  STORE256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:94: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~                                        ^
KeccakP-1600-times4-SIMD256.c:56:70: note: expanded from macro 'STORE256'
    #define STORE256(a, b)          _mm256_store_si256((V256 *)&(a), b)
                                                                     ^
KeccakP-1600-times4-SIMD256.c:238:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                OverWr( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:214:66: note: expanded from macro 'OverWr'
    #define OverWr( argIndex )  STORE256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:55: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~ ^
KeccakP-1600-times4-SIMD256.c:56:70: note: expanded from macro 'STORE256'
    #define STORE256(a, b)          _mm256_store_si256((V256 *)&(a), b)
                                                                     ^
KeccakP-1600-times4-SIMD256.c:238:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                OverWr( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:214:66: note: expanded from macro 'OverWr'
    #define OverWr( argIndex )  STORE256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:68: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~              ^
KeccakP-1600-times4-SIMD256.c:56:70: note: expanded from macro 'STORE256'
    #define STORE256(a, b)          _mm256_store_si256((V256 *)&(a), b)
                                                                     ^
KeccakP-1600-times4-SIMD256.c:238:17: error: implicit conversion changes signedness: 'UINT64' (aka 'unsigned long long') to 'long long' [-Werror,-Wsign-conversion]
                OverWr( i );
                ^~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:214:66: note: expanded from macro 'OverWr'
    #define OverWr( argIndex )  STORE256(stateAsLanes[argIndex], LOAD4_64(curData3[argIndex], curData2[argIndex], curData1[argIndex], curData0[argIndex]))
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
KeccakP-1600-times4-SIMD256.c:50:81: note: expanded from macro 'LOAD4_64'
    #define LOAD4_64(a, b, c, d)    _mm256_set_epi64x((UINT64)(a), (UINT64)(b), (UINT64)(c), (UINT64)(d))
                                    ~~~~~~~~~~~~~~~~~                           ^
KeccakP-1600-times4-SIMD256.c:56:70: note: expanded from macro 'STORE256'
    #define STORE256(a, b)          _mm256_store_si256((V256 *)&(a), b)
                                                                     ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make[2]: *** [KeccakP-1600-times4-SIMD256.o] Error 1
make[1]: *** [../../../common/keccak4x/KeccakP-1600-times4-SIMD256.o] Error 2
make: *** [/Users/ur20980/src/PQClean/testcases/test_testvectors__QA08pBxp/crypto_bla/scheme/impl/libsphincs-shake256-256f-robust_avx2.a] Error 2

================================= short test summary info ==================================
FAILED test_testvectors.py::test_testvectors[avx2 implementation of sphincs-shake256-256s-simple]
FAILED test_testvectors.py::test_testvectors[avx2 implementation of sphincs-shake256-192s-robust]
FAILED test_testvectors.py::test_testvectors[avx2 implementation of sphincs-shake256-128s-simple]
FAILED test_testvectors.py::test_testvectors[avx2 implementation of sphincs-shake256-128s-robust]
FAILED test_testvectors.py::test_testvectors[avx2 implementation of sphincs-shake256-192s-simple]
FAILED test_testvectors.py::test_testvectors[avx2 implementation of sphincs-shake256-256s-robust]
FAILED test_testvectors.py::test_testvectors[avx2 implementation of sphincs-shake256-256f-simple]
FAILED test_testvectors.py::test_testvectors[avx2 implementation of sphincs-shake256-192f-robust]
FAILED test_testvectors.py::test_testvectors[avx2 implementation of dilithium4] - Asserti...
FAILED test_testvectors.py::test_testvectors[avx2 implementation of dilithium3] - Asserti...
FAILED test_testvectors.py::test_testvectors[avx2 implementation of dilithium2] - Asserti...
FAILED test_testvectors.py::test_testvectors[avx2 implementation of sphincs-shake256-128f-simple]
FAILED test_testvectors.py::test_testvectors[avx2 implementation of sphincs-shake256-128f-robust]
FAILED test_testvectors.py::test_testvectors[avx2 implementation of sphincs-shake256-192f-simple]
FAILED test_testvectors.py::test_testvectors[avx2 implementation of sphincs-shake256-256f-robust]
======================== 15 failed, 74 passed in 1341.80s (0:22:21) ========================
```

--- a/common/keccak4x/Makefile
+++ b/common/keccak4x/Makefile
@@ -1,7 +1,7 @@
 KeccakP-1600-times4-SIMD256.o: KeccakP-1600-times4-SIMD256.c \
   align.h brg_endian.h KeccakP-1600-times4-SnP.h \
   KeccakP-1600-unrolling.macros SIMD256-config.h
-	$(CC) $(CFLAGS) -mavx2 -c $< -o $@
+	$(CC) -O3 -mavx2 -c $< -o $@
 
 .PHONY: clean
 clean:

